modules/ROOT/pages/build-application-from-api.adoc

```diff
@@ -0,0 +1,198 @@
+= Build an Application from an API Specification
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: build, application, api, specification
+
+During your app development lifecycle, you can build Mule REST or SOAP APIs from an API specification using APIkit.  You develop APIs based on the following modeling languages:
+
+* RAML (RESTful API Modeling Language)
+* WSDL (Web Service Description Language)
+* OData-Enabled APIs using APIkit OData Extension
+
+By using APIkit in Anypoint Studio, you can simplify the design and implementation of APIs by automating Mule flows creation based on importing either a RAML, or WSDL specification file. After generating the API, you can simulate calls to test the API and then finalize its implementation. +
+
+Additionally, APIkit helps you build applications using Apache Maven, but before you jump into the workflows of building an application using APIkit, learn the basic APIkit anatomy and understand the scaffolding technique that generates Mule flows when you start APIkit projects.
+
+[NOTE]
+MuleSoft recommends Design Center for creating and syncing RAML code; however, Anypoint Studio includes a RAML editor that provides syntax validation and suggestions if you want to perform the entire development within Studio.
```

Comment:
> Anypoint Studio includes a RAML editor that provides syntax validation and suggestions if you want to perform the entire development within Studio. 

I would probably ommit this since the RAML editor is being deprecated.

Reply:
Removed.